### ISSUE TITLE
MAVProxy console settings not working for macOS

### ISSUE THREAD

**Issue by srmainwaring** [ArduPilot/MAVProxy#868]
The The MAVProxy > Settings menu for the console module is not working correctly on macOS.

Steps to replicate:

1. Run SITL:

```bash
$ sim_vehicle.py -v Rover -f rover --map --console
```

2. Select the Console window then select MAVProxy > Settings from the menu bar:

```console
MANUAL> Process Process-5:
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.9/3.9.1_7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/local/Cellar/python@3.9/3.9.1_7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/Volumes/MacPro2_DV1/Code/ardupilot/MAVProxy/MAVProxy/modules/lib/wxsettings.py", line 32, in child_task
    dlg = SettingsDlg(self.settings)
  File "/Volumes/MacPro2_DV1/Code/ardupilot/MAVProxy/MAVProxy/modules/lib/wxsettings_ui.py", line 174, in __init__
    TabbedDialog.__init__(self, tabs, title)
  File "/Volumes/MacPro2_DV1/Code/ardupilot/MAVProxy/MAVProxy/modules/lib/wxsettings_ui.py", line 31, in __init__
    self.dialog_sizer.Add(button_box, 0, wx.GROW|wx.ALIGN_CENTER_VERTICAL|wx.ALL, 5)
wx._core.wxAssertionError: C++ assertion "!(flags & wxALIGN_CENTRE_VERTICAL)" failed at /private/var/folders/s4/4blpy32d4b9_v7_5llt47r7m0000gn/T/pip-install-9ssuf7g3/wxpython/ext/wxWidgets/src/common/sizer.cpp(2077) in DoInsert(): Vertical alignment flags are ignored in vertical sizers
```

**Comment by srmainwaring** [comment]
Removing the wx.ALIGN_CENTER_VERTICAL from the sizers fixes the fault preventing the settings window from opening:

[MAVProxy > Settings] opens the window as expected:

![image](https://user-images.githubusercontent.com/24916364/109688205-eea64500-7b7b-11eb-85ae-488b90e87d25.png)

However there remains a residual warning:

```bash
wxPyDeprecationWarning: Call to deprecated item __call__. Use :meth:`EvtHandler.Bind` instead.
  wx.EVT_BUTTON(self, self.button_cancel.GetId(), self.on_cancel)
/Volumes/MacPro2_DV1/Code/ardupilot/MAVProxy/MAVProxy/modules/lib/wxsettings_ui.py:33: wxPyDeprecationWarning: Call to deprecated item __call__. Use :meth:`EvtHandler.Bind` instead.
  wx.EVT_BUTTON(self, self.button_apply.GetId(), self.on_apply)
/Volumes/MacPro2_DV1/Code/ardupilot/MAVProxy/MAVProxy/modules/lib/wxsettings_ui.py:34: wxPyDeprecationWarning: Call to deprecated item __call__. Use :meth:`EvtHandler.Bind` instead.
  wx.EVT_BUTTON(self, self.button_save.GetId(), self.on_save)
/Volumes/MacPro2_DV1/Code/ardupilot/MAVProxy/MAVProxy/modules/lib/wxsettings_ui.py:35: wxPyDeprecationWarning: Call to deprecated item __call__. Use :meth:`EvtHandler.Bind` instead.
  wx.EVT_BUTTON(self, self.button_load.GetId(), self.on_load)
```

**Comment by srmainwaring** [comment]
Closed by #871
